js/views/messagesitem.js

```diff
@@ -102,7 +109,16 @@ define(function(require) {
 				revert: 'invalid'
 			});
 
-			tooltip('.action.delete', {placement: 'left'});
+			// TODO: Close mail actions menu on outside click
```

Comment:
there's some logic for that. you'll find it at other code sections where we open menus as well. let me know if you can't find it :)

Reply:
I already found it and implemented it. I just forgot to remove the comment 😅

Reply:
hehe, good :grinning: